tracing/os-tracing/README.md

```diff
@@ -0,0 +1,24 @@
+## OS Tracing  
+
+* [DTrace](http://dtrace.org/)  
+  Available on IllumOS, Solaris, Mac
+  The [DTrace blog](http://dtrace.org/blogs/blog/category/node-js/) has some articles on using DTrace with Node.js
+
+* [LTTng](http://lttng.org/)  
+  Linux only.  
+  Some info on using LTTng with Node.js are available in the presentations
+  [Tracing Node on Linux with LTTNG](http://nearform.github.io/tracing-node-lttng-nodejsdublin/)
```

Comment:
An example of a Node.js application which implements LTTng tracepoints can be found here: https://github.com/thekemkid/magic-tracing

Quick demo of tracing using it: https://asciinema.org/a/16785